### DISCUSSION TITLE
Using along with Ktor

### DISCUSSION THREAD

**Discussion by mildroid** [nacular/doodle#57] [Ideas]
Hi there.
An amazing work has done with this library💪.
I wonder if it's possible to draw Ktor web app's UI with Doodle?
Thanks.

**Comment by pusolito** [comment]
Hey. This is definitely possible. The [Photo Stream](https://nacular.github.io/doodle-tutorials/docs/photostream) tutorial actually uses Ktor to fetch splash images.

**Comment by mildroid** [comment]
That's cool man, Keep it up.
